vexcl/optimisation.hpp

```diff
@@ -0,0 +1,200 @@
+#ifndef OPTIMISATIONS_HPP
+#define OPTIMISATIONS_HPP
+
+
```

Comment:
Can you add a copyright notice here, similar to the rest of the header files? (you should put your name there).